src/Components/test/E2ETest/Tests/AuthTest.cs

```diff
@@ -12,7 +13,7 @@
 namespace Microsoft.AspNetCore.Components.E2ETest.Tests
 {
     [Collection("auth")] // Because auth uses cookies, this can't run in parallel with other auth tests
-    public class AuthTest : BasicTestAppTestBase
```

Comment:
I expected to see more files moving around since we were going to organize tests by which startup they use? Not part of this PR?

Reply:
Not part of this PR. I'm trying to this in steps and this is already a pretty big PR